runtime/src/alpha/mod.rs

```diff
@@ -172,9 +201,9 @@ where
         self,
         factory: Fac,
         resource_tracker: ResourceTracker,
-        logger: Logger,
+        deployment_id: String,
```

Comment:
Is this eventually used?

Reply:
It's not used anymore, removed in https://github.com/shuttle-hq/shuttle/pull/1226/commits/89be0ec06c86809d3752a52231d95710253a3f04